src/normalize/normalize.py

```diff
@@ -0,0 +1,109 @@
+            alerts.append((entity_id, entity.alert))
+        if entity.HasField('vehicle'):
+            vehicles.append((entity_id, entity.vehicle))
+
+    trip_updates_pa = protobuf_objects_to_pyarrow_table([x[1] for x in trip_updates]) if trip_updates else None
+    vehicles_pa = protobuf_objects_to_pyarrow_table([x[1] for x in vehicles]) if vehicles else None
+    alerts_pa = protobuf_objects_to_pyarrow_table([x[1] for x in alerts]) if alerts else None
+
+    if trip_updates_pa:
+        trip_updates_pa = trip_updates_pa.add_column(0, "id", [[x[0] for x in trip_updates]])
+        trip_updates_pa = add_time_columns(trip_updates_pa, cur_time, cur_date)
+        s3_uri = f"{output_prefix}/trip-updates"
+        write_data(trip_updates_pa, s3_uri)
+
+    if vehicles_pa:
+        vehicles_pa = vehicles_pa.add_column(0, "id", [[x[0] for x in vehicles]])
+        vehicles_pa = add_time_columns(vehicles_pa, cur_time, cur_date)
+        s3_uri = f"{output_prefix}/vehicles"
+        write_data(vehicles_pa, s3_uri)
+
+    if alerts_pa:
+        alerts_pa = alerts_pa.add_column(0, "id", [[x[0] for x in alerts]])
+        alerts_pa = add_time_columns(alerts_pa, cur_time, cur_date)
+        s3_uri = f"{output_prefix}/alerts"
+        write_data(alerts_pa, s3_uri)
+
+
+def parse_files(source_prefix, destination_prefix, state_file):
+    s3 = boto3.client('s3')
+
+    source_bucket, source_key = source_prefix.replace("s3://", "").split("/", 1)
+    state_bucket, state_key = state_file.replace("s3://", "").split("/", 1)
+
+    last_processed = get_last_processed_timestamp(s3, state_bucket, state_key)
+
+    # List objects in the source bucket
+    paginator = s3.get_paginator('list_objects_v2')
+
+    for page in paginator.paginate(Bucket=source_bucket, Prefix=source_key):
+        for obj in page.get('Contents', []):
+            key = obj['Key']
+            if key.endswith('.bin'):
+                # Check if this file is newer than the last processed file
+                if last_processed and obj['LastModified'].timestamp() <= float(last_processed):
+                    continue
+
+                # Download the file
+                response = s3.get_object(Bucket=source_bucket, Key=key)
+                file_content = response['Body'].read()
+
+                normalize_raw_feed(
+                    file_content,
+                    destination_prefix,
+                    obj['LastModified'],
+                    obj['LastModified'].date())
+
+                # Update the last processed timestamp
+                update_last_processed_timestamp(s3, state_bucket, state_key, str(obj['LastModified'].timestamp()))
+
+
+@click.command()
+@click.option('--source-prefix', required=True, help='Source S3 prefix (e.g., s3://bucket/prefix)')
+@click.option('--destination-prefix', required=True, help='Destination S3 prefix for Parquet files')
+@click.option('--state-file', required=True, help='S3 path to store the state (e.g., s3://bucket/state.json)')
```

Comment:
Can we stuff these config to `config/config.json` so that the only required argv for normalizer will be `-f FEED_ID`. This would help align the CLI for both scraping and normalizing.